examples/plot_decoding_tutorial.py

```diff
@@ -281,8 +300,10 @@
 #
 # We can plot the weights, using the subject's anatomical as a background
 from nilearn.plotting import plot_stat_map, show
-plot_stat_map(decoder.coef_img_['face'], bg_img=haxby_dataset.anat[0],
-              title="SVM weights", display_mode="yx")
+plot_stat_map(
```

Comment:
I think that I'd like us to use view_img here: to get the interactive display, as this is what most people need.

Reply:
Yes I miss  Should be interactive like the visualizing fmri volume section above.